src/components/HyperlinkModal.jsx

```diff
@@ -18,45 +28,44 @@ const HyperlinkModal = ({ onSave, initialText, onClose }) => {
   const [link, setLink] = useState("")
 
   return (
-    <>
-      <div className={elementStyles.overlay}>
-        <div className={elementStyles["modal-settings"]}>
-          <div className={elementStyles.modalHeader}>
-            <h1>Insert hyperlink</h1>
-            <CloseButton onClick={onClose} />
-          </div>
-          <div className={elementStyles.modalContent}>
-            <div>
-              <FormContext isRequired>
-                <VStack display="block">
-                  <Box>
-                    <FormTitle>Text</FormTitle>
-                    <FormField
-                      placeholder="Text"
-                      id="text"
-                      value={text}
-                      onChange={(event) => setText(event.target.value)}
-                    />
-                  </Box>
-                  <Box>
-                    <FormTitle>Link</FormTitle>
-                    <FormField
-                      placeholder="Link"
-                      id="link"
-                      value={link}
-                      onChange={(event) => setLink(event.target.value)}
-                    />
-                  </Box>
-                  <Flex justifyContent="end">
-                    <Button onClick={() => onSave(text, link)}>Save</Button>
-                  </Flex>
-                </VStack>
-              </FormContext>
-            </div>
-          </div>
-        </div>
-      </div>
-    </>
+    <Modal isOpen onClose={onClose}>
```

Comment:
no real change in functionality, just swapped to chakra stuff; rest is indentation